app/src/views/Main.vue

```diff
@@ -295,18 +310,21 @@ export default {
               }
             }
           });
-          console.log(res);
         } catch (error) {
-          console.log(error);
-          alert("Something went wrong, please refresh");
+          this.handleError(error);
         }
       } else {
         console.log("just passing for now");
       }
 
-      console.log(this.recs[this.event_id][0]);
+      // apparently has to be done this way for Vue to react to nested stuff
```

Comment:
Probably since object comparison is done on the instance https://vuejs.org/v2/guide/reactivity.html#For-Objects